src/dataStrategies.js

```diff
@@ -36,6 +37,7 @@ export const results = ({
         sortDir,
         scroll: true,
         scrollId,
+        highlight: !disableHighlight
```

Comment:
results actually is fine with undefined, so you should just pass along `highlight` directly instead of inverting it.